client/src/components/Ratings/sub-components/ReviewList.jsx

```diff
@@ -13,22 +14,42 @@ export default function ReviewList({
   setListIndex,
   feature,
   reviewMeta,
+  setReviewMeta,
 }) {
   // STATE DATA //
   const [reviewExpander, setReviewExpander] = useState('25rem');
   const [expandedStatus, setExpandedStatus] = useState(false);
   const [reviewModal, setReviewModal] = useState(false);
+  const [searchTerm, setSearchTerm] = useState('');
 
   // HELPER FUNCTIONS //
-  const reviewMapper = (reviewArray) => reviewArray.map((review) => (
-    <ReviewTile
-      review={review}
-      key={review.review_id}
-      reviews={reviews}
-      setReviews={setReviews}
-      feature={feature}
-    />
-  ));
+
+  const getDateString = (dateString) => {
+    const date = new Date(dateString);
+    const months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+    const monthName = months[date.getMonth()];
+    const day = date.getDate();
+    const year = date.getFullYear();
+    return `${monthName} ${day}, ${year}`;
+  };
+
+  const reviewMapper = (reviewArray) => reviewArray.map((review) => {
+    if (review.body.toLowerCase()
```

Comment:
please add a TODO here for us to revisit. I think we can abstract this out and make this a little more readable. your search logic seems like it should be pretty similar to mine.